### ISSUE TITLE
Lisu: the advance width of some tone marks are wrong

### ISSUE THREAD

**Issue by jungshik** [googlei18n/noto-fonts#466]
Imported from [Google Code issue #244](https://code.google.com/a/google.com/p/noto-alpha/issues/detail?id=244) created by [roozbeh@google.com](https://code.google.com/u/roozbeh@google.com/) on 2014-06-14T03:04:25.000Z:

----
From the text of the Unicode Standard, the section on Lisu, and also from the proposal documents about Lisu, such as http://www.unicode.org/L2/L2008/08019-lisu.pdf, it appears that the tone marks that look like single European punctuation have a lot of white space after them. But the current Noto Sans Lisu font treats the characters the same as European punctuation and centers them with small advance widths.

/cc @kmansourMT

**Comment by nizarsq** [comment]
Looked at the following tone marks, hopefully I got the right marks.
F4FA 
A4F9 ꓹ
A4FA ꓺ
A4FB ꓻ
A4FC ꓼ
A4FD ꓽ 
Look like they still don't have enough white space.
<img width="937" alt="Screen Shot 2020-07-29 at 8 11 53 PM" src="https://user-images.githubusercontent.com/22158882/88876272-c286a800-d1d7-11ea-8813-d493ce7ae214.png">

**Comment by dscorbett** [comment]
I don’t think this report is accurate. The first post cites [L2/08-019](https://www.unicode.org/L2/L2008/08019-lisu.pdf), but there are many examples in that proposal of tone marks without white space after them. The white space that appears after syllables with tone marks is the same white space that appears after syllables _without_ tone marks, i.e. U+0020 SPACE, not part of the tone mark glyphs.

In [figure 1](https://www.unicode.org/L2/L2008/08019-lisu.pdf#page=20), in verse 1, there is “ꓚꓰꓼˍ”. In [figure 4](https://www.unicode.org/L2/L2008/08019-lisu.pdf#page=22), in line 4 of item 2.ꓐ, there is “ꓕꓯꓽ꓾”. In [figure 5](https://www.unicode.org/L2/L2008/08019-lisu.pdf#page=22), there are many circled examples of combination tones. In [figure 9](https://www.unicode.org/L2/L2008/08019-lisu.pdf#page=25), in the black circle, there is “ꓡꓳꓺꓽ꓿”. In none of these examples is there any space after non-final tone-marks.
